diff --git a/cmake/FindMAVLink.cmake b/cmake/FindMAVLink.cmake
index 8de3079..c203237 100644
--- a/cmake/FindMAVLink.cmake
+++ b/cmake/FindMAVLink.cmake
@@ -46,13 +46,6 @@ find_path(_MAVLINK_INCLUDE_DIR
     NO_CMAKE_SYSTEM_PATH
     )
 
-# look specifically for the ROS version if no other was found
-find_path(_MAVLINK_INCLUDE_DIR
-   NAMES mavlink/v1.0/mavlink_types.h mavlink/v2.0/mavlink_types.h
-   PATH_SUFFIXES include
-   PATHS /opt/ros/${ROS_DISTRO}/
-   )
-
 # read the version
 if (EXISTS ${_MAVLINK_INCLUDE_DIR}/mavlink/config.h)
     file(READ ${_MAVLINK_INCLUDE_DIR}/mavlink/config.h MAVLINK_CONFIG_FILE)
diff --git a/package.xml b/package.xml
index ae0fb34..12f17b8 100644
--- a/package.xml
+++ b/package.xml
@@ -50,8 +50,8 @@
   <build_depend>roscpp</build_depend>
   <build_depend>sensor_msgs</build_depend>
   <build_depend>std_msgs</build_depend>
-  <build_depend>python-rospkg</build_depend>
-  <build_depend>python-jinja2</build_depend>
+  <build_depend>python3-rospkg</build_depend>
+  <build_depend>python3-jinja2</build_depend>
   <run_depend>eigen</run_depend>
   <run_depend>gazebo_ros</run_depend>
   <run_depend>geometry_msgs</run_depend>
